packages/graphql/src/translate/authorization/utils/filter-by-values.ts

```diff
@@ -42,7 +43,13 @@ export function filterByValues<T>(
             }
         } else {
             const { fieldName, operator } = parseFilterProperty(k);
-            const receivedValue = receivedValues[fieldName];
+            let receivedValue: T | undefined;
+            const mappedJwtClaim = jwtClaims?.get(fieldName);
+            if (mappedJwtClaim) {
+                receivedValue = expandJwtClaim(mappedJwtClaim, receivedValues);
+            } else {
+                receivedValue = receivedValues[fieldName];
+            }
```

Comment:
Nitpick

This could be a separate method to get the `receivedValue` from a `fieldName`